rstudio-wsl/draft.Rmd

```diff
@@ -0,0 +1,157 @@
+
+You can now close the WSL instance by typing the following command:
+   
+```{bash}
+exit
+```
+
+* **Note:** Unlike the PowerShell, you cannot use `Ctrl + V` shortcut key to paste the clipboard content into the WSL terminal. Alternatively, press mouse right-click button to emulate the paste function.
+
+## RStudio Server Setup {#rstudio-server-setup}
+
+Now that you have successfully installed a Linux distribution on your system via WSL, you can setup and run RStudio Server by following the procedures below:
+
+1. Type `wsl` in either PowerShell or CMD terminal to launch your installed linux distro.
+1. Make sure your Linux packages up-to-date using the following command (we will assume that you have installed a Ubuntu distribution):
+
+    ```{bash}
+    sudo apt-get update
+    sudo apt-get upgrade -y
+    ```
+
+    * **Note:** Provide the password of your user account that you created for the Linux system, not the Windows password to log in for your PC.
+
+1. Add the following apt credential to your repository list  for Getting the Latest R (v4.0.0 as of this writing):
+
+    ```{bash}
+    sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
+    sudo add-apt-repository 'deb https://cloud.r-project.org/bin/linux/ubuntu focal-cran40/'
+    ```
+
+    * **Note:** More detailed information on installing the latest R for Linux can be found at [the official CRAN page](https://cran.r-project.org/bin/linux/ubuntu/){target="_blank"}.
+
+1. Install R with some essential dependencies for the RStudio Server and devtools:
+
+    ```{bash}
+    sudo apt install -y r-base r-base-core r-recommended r-base-dev gdebi-core build-essential libcurl4-gnutls-dev libxml2-dev libssl-dev
+    ```
+
+1. Obtain the download URL of the RStudio Server you want to install.
+
+    * **Note:** You can explore some RStudio download pages, such as [RStudio Release](https://rstudio.com/products/rstudio/download/#download){target="_blank"}, [RStudio Preview (close to release)](https://rstudio.com/products/rstudio/download/preview/){target="_blank"}, and [RStudio Daily](https://dailies.rstudio.com/){target="_blank"}. Make sure you choose RStudio *Server* (not Desktop) and match with your installed Linux distribution type and version.
+
+1. Install RStudio Server:
+
+    ```{bash}
+    wget https://s3.amazonaws.com/rstudio-ide-build/server/bionic/amd64/rstudio-server-1.4.467-amd64.deb
+    sudo gdebi rstudio-server-1.4.467-amd64.deb
+    ```
+
+    * **Note1:**We have downloaded the daily version, `rstudio-server-1.4.467-amd64.deb` for this demonstration; however, the URL and file name can vary depending on what you have chosen.
+
+    * **Note2:** You may be prompted a warning message like "Couldn't find an alternative telinit implementation to spawn." However, our testing results have revealed that there is no issue with using RStudio Server regardless of this warning.
```

Comment:
change: "You may be prompted a warning message" to "You may receive a warning message"

Reply:
What this message actually means is that RStudio Server will not automatically start when the Linux environment starts. So mention that if you terminate and restart WSL you will need to run `sudo rstudio-server start` again to get RStudio Server up and running. I haven't investigated to see if we can work around this but hopefully we can eventually (don't mention that in the article).